docs/v3/PrintOne.md

```
@@ -0,0 +1,275 @@
+
+> Note: `client.v3Client` throws if the client was not initialized with `{ version: "v3" }`.
+
+---
+
+# Methods (v3)
+
+## `.getCampaigns([options])`
+
+Get all campaigns.
+
+**Parameters**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)